test/e2e-common.sh

```diff
@@ -301,6 +302,10 @@ function install() {
     YTT_FILES+=("${REPO_ROOT_DIR}/test/config/ytt/mesh")
   fi
 
+  if (( AMBIENT )); then
```

Comment:
Could you fix this one? Probably needs not space: `if ((AMBIENT)); then`

Reply:
I think we should defer to https://github.com/knative/serving/pull/13971 to keep consistency as I'm not sure if https://github.com/knative/serving/pull/13971 will fix them.